platform/darwin/src/MGLLight.h

```diff
@@ -0,0 +1,66 @@
+#import <CoreGraphics/CoreGraphics.h>
+
+#import "MGLFoundation.h"
+#import "MGLStyleValue.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSUInteger, MGLLightAnchorType) {
+    MGLLightAnchorTypeMap,
+    MGLLightAnchorTypeViewport
+};
+
+MGL_EXPORT
+@interface MGLLight : NSObject
+
+/**
+ `lightAnchorType` property.
+ */
+@property (nonatomic) MGLStyleValue<NSValue *> *lightAnchorType;
```

Comment:
Judging from [the style specification documentation](https://www.mapbox.com/mapbox-gl-js/style-spec/#light-anchor), the `anchor` property can’t be set to a function and isn’t transitionable. So the type of this property should be simply `MGLLightAnchorType`, and the `lightAnchorTypeTransition` property should be removed.